### PR TITLE
#22328 Fix Procedures are hidden in navigator if any function exist in a schema

### PR DESCRIPTION
**Pull request by Stkai**
#22328 Fix Procedures are hidden in navigator if any function exist in a schema
#22329 Fix Exec keywords

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ext.dameng/src/org/jkiss/dbeaver/ext/dameng/model/DamengMetaModel.java [hunk 1]
```diff
@@ -227,9 +231,71 @@ public List<? extends GenericTrigger> loadTriggers(DBRProgressMonitor monitor, G
...
+                            "PROC_OBJ.INFO1,\n" +
+                            "PROC_OBJ.VALID\n" +
+                            "FROM\n" +
+                            "SYSOBJECTS PROC_OBJ,\n" +
+                            "SYSOBJECTS SCH_OBJ,\n" +
+                            "SYSOBJECTS USER_OBJ\n" +
+                            "WHERE\n" +
+                            "PROC_OBJ.SCHID = SCH_OBJ.ID\n" +
+                            "AND SCH_OBJ.PID = USER_OBJ.ID\n" +
+                            "AND PROC_OBJ.SUBTYPE$ = 'PROC'\n" +
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.ext.dameng/src/org/jkiss/dbeaver/ext/dameng/model/DamengMetaModel.java:252]: `PROC_OBJ.SUBTYPE$ = 'PROC'`

Does it mean that functions will be also in the query result?

Comment (Stkai) [reply plugins/org.jkiss.dbeaver.ext.dameng/src/org/jkiss/dbeaver/ext/dameng/model/DamengMetaModel.java:252]: Yes, contains function , distinguished by info1

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.ext.dameng/src/org/jkiss/dbeaver/ext/dameng/model/DamengMetaModel.java:252]: Ok

**Review by LonwoLonwo — APPROVED** [review]

**Comment by Matvey16** [comment]
Verified

**Pull request merged**